modules/ensembl/metadata/genomesummary/main.nf

```diff
@@ -0,0 +1,55 @@
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+process METADATA_GENOMESUMMARY {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "ensemblorg/ensembl-genomio:v1.6.1"
+
+    input:
+        tuple val(meta), path(input_query), path(genome_summary_json)
+
+    output:
+        tuple val(meta), path("genome.json"), emit: meta_json
+        path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
```

Comment:
Tab alignment missing?

Reply:
So you would suggest we entirely separate out the licence header to a different LICENSE file. Im open to this approach, I would agree its much cleaner. Though I believe we are still required to have this snippet at the top. I will raise with Jorge and see, thank you for your input 👍